lib/Runtime/ByteCode/ScopeInfo.h

```diff
@@ -29,6 +29,7 @@ namespace Js {
             Field(SymbolType) symbolType;
             Field(bool) hasFuncAssignment;
             Field(bool) isBlockVariable;
+            Field(bool) isConst;
             Field(bool) isFuncExpr;
             Field(bool) isModuleExportStorage;
             Field(bool) isModuleImport;
```

Comment:
Can we make these bit fields. Saves 4 bytes on x86.

Reply:
Sure.